java/vector/pom.xml

```diff
@@ -122,12 +119,6 @@
             <goals>
               <goal>test</goal>
             </goals>
-            <configuration>
-              <classpathDependencyExcludes>
-                <classpathDependencyExclude>org.apache.arrow:arrow-memory-netty</classpathDependencyExclude>
-              </classpathDependencyExcludes>
-              <reportNameSuffix>netty</reportNameSuffix>
-            </configuration>
```

Comment:
I think this should be added back

Reply:
The changes have been reverted in https://github.com/sirensolutions/arrow/pull/50/commits/fb7d4277d4f61340b3674c375dbf40e00daf7b5e